### ISSUE TITLE
Support URL for library imports

### ISSUE THREAD

**Issue by zaibon** [Jumpscale/go-raml#282]
Currently go-raml doesn't seems to support URL when importing libraries.

example raml:
```raml
uses:
  reservation: https://raw.githubusercontent.com/gig-cockpit/raml-definitions/master/librairies/reservation.raml
```

error : 
```
ERRO[2017-05-18T11:30:23+04:00] apiDef.PostProcess() failed to parse library	name=reservation, path=https://raw.githubusercontent.com/gig-cockpit/raml-definitions/master/librairies/reservation.raml
	err=Could not read file ../specs/raml/https:/raw.githubusercontent.com/gig-cockpit/raml-definitions/master/librairies/reservation.raml (Error: open ../specs/raml/https:/raw.githubusercontent.com/gig-cockpit/raml-definitions/master/librairies/reservation.raml: no such file or directory) 

```

**Comment by iwanbk** [comment]
A work in progress can be found  on `libraries-in-file` (just realized the bad branch name) branch.
It updated the parser to support URL for library imports. Only need to fix the generator.

**Comment by iwanbk** [comment]
@zaibon fixed by #291 

Please note that libraries support only available on Go server & client.
There is beta support for flask server.

Please create issue if you need another variant (client/server/language) support.
